projects/addon-editor/components/editor/editor.component.ts

```diff
@@ -49,7 +50,11 @@ import {TUI_EDITOR_PROVIDERS} from './editor.providers';
     templateUrl: './editor.component.html',
     styleUrls: ['./editor.component.less'],
     changeDetection: ChangeDetectionStrategy.OnPush,
-    providers: [tuiAsFocusableItemAccessor(TuiEditorComponent), TUI_EDITOR_PROVIDERS],
+    providers: [
+        tuiAsFocusableItemAccessor(TuiEditorComponent),
+        tuiAutoFocusOptionsProvider({delay: 300}),
```

Comment:
What does this magic number `300` means ?
`TUI_ANIMATIONS_DURATION` ?

Reply:
in fact, 300 ms is an inaccurate time anyway, because sometimes `div[contenteditable]` doesn’t have time to render there